maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,86 @@
+# ${artifactId}
+
+#[[
+## Designing
+]]#
+While designing your service it is useful to read [designing services](https://developer.lightbend.com/docs/akka-serverless/designing/index.html)
+
+#[[
+## Developing
+]]#
+This project has a bare-bones skeleton service ready to go, but in order to adapt and
+extend it it may be useful to read up on [developing services](https://developer.lightbend.com/docs/akka-serverless/developing/index.html)
+and in particular the [Java section](https://developer.lightbend.com/docs/akka-serverless/java-services/index.html)
+
+#[[
+## Building
+]]#
+To build, at a minimum you need to generate and process sources, particularly when using an IDE.
+A convenience is compile your project:
+
+```
+mvn compile
+```
+
+#[[
+## Running Locally
+]]#
+In order to run your application locally, you must run the Akka Serverless proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Akka Serverless proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+### macOS and Windows
+
+```
+docker-compose up
+```
+
+### Linux
+
+> On Linux this requires Docker 20.10 or later (https://github.com/moby/moby/pull/40007),
+> or for a `USER_FUNCTION_HOST` environment variable to be set manually.
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.linux.yml up
+```
+
+To start the application locally, the `exec-maven-plugin` is used. Use the following command:
+
+```
+mvn compile exec:java
+```
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://developer.lightbend.com/docs/akka-serverless/java/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
+
+```
+> curl -XPOST -H "Content-Type: application/json" localhost:9000/${package}.MyServiceEntity/GetValue -d '{"entityId": "foo"}'
+The command handler for `GetValue` is not implemented, yet
+```
+
+For example, given [`grpcurl`](https://github.com/fullstorydev/grpcurl):
```

Comment:
"given" --> "using"  perhaps?

(Ditto in maven-java/akkaserverless-maven-archetype-value-entity/src/main/resources/archetype-resources/README.md)

Reply:
```suggestion
For example, using [`grpcurl`](https://github.com/fullstorydev/grpcurl):
```